pkgs/top-level/python-packages.nix

```diff
@@ -9030,6 +9038,10 @@ in {
 
   weasyprint = callPackage ../development/python-modules/weasyprint { };
 
+  webargs = callPackage ../development/python-modules/webargs { };
+
+  webargs5 = callPackage ../development/python-modules/webargs/5.nix { };
```

Comment:
Please use a pythonOverride instead.

Reply:
It's not about a different python

Reply:
You could do it like this:

```diff
diff --git a/pkgs/development/python-modules/aiohttp-apispec/default.nix b/pkgs/development/python-modules/aiohttp-apispec/default.nix
index 31783867456..31fa0d122fc 100644
--- a/pkgs/development/python-modules/aiohttp-apispec/default.nix
+++ b/pkgs/development/python-modules/aiohttp-apispec/default.nix
@@ -2,6 +2,16 @@
 , aiohttp, webargs, apispec3, jinja2
 }:
 
+let
+  webargs5 = webargs.overridePythonAttrs (oldAttrs: rec {
+    version = "5.5.3";
+    src = oldAttrs.src.override {
+      inherit version;
+      sha256 = "16pjzc265yx579ijz5scffyfd1vsmi87fdcgnzaj2by6w2i445l7";
+    };
+    doCheck = false;
+  });
+in
 buildPythonPackage rec {
   pname = "aiohttp-apispec";
   version = "2.2.1";
@@ -12,7 +22,7 @@ buildPythonPackage rec {
     sha256 = "0hhlmh3mc3xg68znsxyhypb5k12vg59yf72qkyw6ahg8zy3qfz2m";
   };
 
-  propagatedBuildInputs = [ aiohttp webargs apispec3 jinja2 ];
+  propagatedBuildInputs = [ aiohttp webargs5 apispec3 jinja2 ];
 
   # No idea how to run the tests
   doCheck = false;
```

Of course the `doCheck = false` is still problematic.

Reply:
I leave it as is, clarifying that there are no tests in the pypi archive.

Reply:
> I leave it as is, clarifying that there are no tests in the pypi archive.

Just do fetchFromGitHub.

> You could do it like this:

It should be overwritten in the application and only exist there.

Reply:
I tried again to run the webargs 5 tests, and I reached the point until they run but one fails: webapp2 related. Because of a python3 problem with webapp2 that is what makes webapp2 have doCheck=false too.

So do I really have to put aiohttp-apispec and webargs5 within the tribler expression, at the end?

Reply:
Another way would be callPackage as overrideAttrs doesn't work too well with python modules